Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -365,12 +366,23 @@ def longFileLinter : Linter where run := withSetOptionIn fun stx ↦ do
     -- `candidate` is necessarily bigger than `lastLine` and hence bigger than `defValue`
     let candidate := (lastLine / 100) * 100 + 200
     let candidate := max candidate defValue
-    if linterBound < lastLine then
+    -- in this case, the file is longer than the default and also than what the option says
+    if defValue ≤ linterBound && linterBound < lastLine then
```

Comment:
It seems we have a test for this, when the default value is exceeded. Can you also add one where it's equal? (I don't want to have to worry about off-by-one errors, but have a test having my back.)

Reply:
Fair enough: I'm also won't object to not verifying that situation super closely.